Resources/public/js/external/ez-translator.js

```diff
@@ -0,0 +1,8 @@
+/* global Translator */
```

Comment:
The standard doc comment is missing. Also, `Y.eZ.Translator` deserves a small doc block explaining what it is (see It deserves a doc block to explain what it is, see https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/external/ez-capi.js#L11 for an example).